sass/components/_popover.scss

```diff
@@ -5,9 +5,11 @@
   position: absolute;
   min-width: 10px;
   opacity: 0;
+  display: none;
```

Comment:
Sky, I have to set display: none; because even if the popover is not displaying it still clickable. Please refer to this video: http://screencast.com/t/j1nCHkgvVG